test/e2e_node/conformance/run_test.sh

```diff
@@ -134,14 +134,14 @@ wait_kubelet() {
   local cur=1
   while [ $cur -le $maxRetry ]; do
     if curl -s $healthCheckURL > /dev/null; then
-      echo "Kubelet is ready"
+      echo "kubelet is ready"
```

Comment:
This one and below are also info messages, but I'm not sure if they should be capitalized or not. Can you look how it's done in other shell scripts?

Reply:
I think `kubelet` is a fixed noun, and there is no problem with the whole lowercase of this noun

Reply:
That's one point of view. Another one is that info messages should be capitalized. You can see them capitalized in [the same file](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/conformance/run_test.sh#L119)

I'd suggest to keep them capitalized.